src/io/flutter/logging/FlutterLogView.java

```diff
@@ -428,8 +459,55 @@ public FlutterLogTreeTable(@NotNull FlutterLogTreeTableModel model) {
     @SuppressWarnings("EmptyMethod")
     @Override
     public TableCellRenderer getCellRenderer(int row, int column) {
-      // TODO(pq): add cell renderer
+      // TODO(pq): do this more robustly
+      if (column == 2) {
```

Comment:
What's the state of the art here? Do other table models have hard-coded column ==> cell renders logic? If so, a named constant here may help this code when other people refacgtor.

Reply:
👍 Looking into just this.  There are a few approaches in the wild but still need to understand the ramifications.  Needless to say, definitely something better than `column == 2` in short order for sure!